### PR TITLE
Added A New User Handler and create a new Go file

### PR DESCRIPTION
**Pull request by Demoncrush44**
Created New User Handler would like a review on any necessary changes

### CODE REVIEW

File: api.go
Comment (kphan32) [inline api.go:1]: rename this file to something like `newuser.go`

File: api.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	ID string `json:"id"`
+}
+
+func handleNewUser(db *DB) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// Check if the request method is POST
+		if r.Method != http.MethodPost {
+			http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+			return
+		}
```
Comment (kphan32) [inline api.go:23]: since ur doing `r.Post` to route it, this probably isnt necessary

**Review by kphan32 — CHANGES_REQUESTED** [review]
lgtm pending file name change

**Review by Demoncrush44 — COMMENTED** [review]
Yes sir

**Review by kphan32 — APPROVED** [review]

**Pull request merged**